elements/bulbs-truncate-element/bulbs-truncate-element.js

```diff
@@ -0,0 +1,38 @@
+import util from 'bulbs-elements/util';
+import {
+  registerElement,
+  BulbsHTMLElement,
+} from 'bulbs-elements/register';
+import './bulbs-truncate-element.scss';
+
+class BulbsTruncateElement extends BulbsHTMLElement {
+
+  attachedCallback () {
+    this.addButton = this.addButton.bind(this);
+    this.openElement = this.openElement.bind(this);
+    this.addButton();
+    this.addEventListener('click', this.openElement);
+    $(this).prev().addClass('bulbs-truncate-element-parent-active');
```

Comment:
Good use of native dom methods in this element, another two apis we can use here

`this.previousElementSibling`

and `element.classList`

Together we can use them like:

```js
let previousEl = this.previousElementSibling;
if (previousEl) {
  previousEl.classList.add('bulbs-truncate-element-parent-active');
  previousEl.classList.remove('bulbs-truncate-element-parent-active');
}
```

Doing this now will help when we start migrating these components over to the `gawkermedia/kinja-components` repository which does not include jQuery as a depency.